features/itunes/components/TrackList/index.tsx

```diff
@@ -74,40 +60,11 @@ const TrackList: React.FC<TrackListProps> = props => {
                 sm={{ span: 12 }}
                 md={{ span: 8 }}
               >
-                <CustomTrackCard data-testid="track-card">
-                  <AlbumArt data-testid="album-art" src={item.artworkUrl100}></AlbumArt>
-                  <TrackName>
-                    <BlockText id="track_name" values={{ name: !isUndefined(item.trackName) && item.trackName.substring(0, 10) }} />
-                  </TrackName>
-                  <ArtistName>
-                    <BlockText
-                      id="artist_name"
-                      values={{ name: item.artistName.substring(0, 10) }}
-                    />
-                  </ArtistName>
-                  <Duration>
-                    <BlockText id="duration" />
-                    {Math.floor(item.trackTimeMillis / 60000)}:
-                    {Math.floor((item.trackTimeMillis / 1000) % 60)}s
-                  </Duration>
-                  <audio
-                    id={item.previewUrl}
-                    autoPlay={false}
-                    controls
-                    onPlay={() => trackEventHandler(item.trackId, item.previewUrl)}
-                    ref={memoizedAudioRef ? memoizedAudioRef?.[item.trackId] : null}
-                    style={
-                      {
-                        display: "block",
-                        width: "fit-content<100%>",
-                        maxWidth: "100%",
-                      } as React.CSSProperties
-                    }
-                  >
-                    <source src={item.previewUrl} type="audio/mp3"></source>
-                    Your browser does not support audio tags
-                  </audio>
-                </CustomTrackCard>
+                <TrackCard
```

Comment:
Do we have a shimmer while the tracks are loading ?

Reply:
like skeleton tag from antd?